src/test/scala/edu/ie3/simona/agent/grid/ThermalGridIT.scala

```diff
@@ -996,7 +997,9 @@ class ThermalGridIT
                 emResult._3 should equalWithTolerance(
                   -0.002517561515.asMegaWatt
                 )
-                emResult._4 should equalWithTolerance(0.0012489996.asMegaVar)
+                emResult._4 should equalWithTolerance(
+                  -0.00082748245392177.asMegaVar
+                )
```

Comment:
Also this looks like an entirely different value...

Reply:
That one is caused by line 74-75 where I changed the tolerance. Which was a dangerous one we missed in the previous PRs. I can fix that in a separate PR if you like for better traceability?

Reply:
Well the difference between `1.2489996 kVar` and `-0.82748245392177 kVar` surely seems to be _not_ caused by a changed testing tolerance, right?

Reply:
Ah, ok, I see. It was the `0.01` implicit double.